lib/addRemove.js

```diff
@@ -64,7 +64,7 @@ function addAsync(version, useNow) {
 			let versionDir = nvsUse.getVersionDir(version);
 			removeDirectoryRecursive(versionDir);
 
-			let remoteUri = settings.remotes[version.remoteName];
+			let remoteUri = version.uri;
 			return downloadAndExtractAsync(version, remoteUri).then(() => {
```

Comment:
You are right. My initial changes were based on older version of nvs (`0.x` IIRC). The change was to fix the parsing of extension correctly. Since you must have already fixed it, I will remove the unwanted `remoteUri`.